AUR/PKGBUILD

```diff
@@ -2,39 +2,38 @@
 # Maintainer: Antoine Bertin <ant.bertin@gmail.com>
 # Maintainer: Andrey Kolchenko <andrey@kolchenko.me>
 
-pkgname=linux-enable-ir-emitter
-pkgver=20210725.0
+pkgname=linux-enable-ir-emitter-git
+pkgver=r167.04f6c73
 pkgrel=1
 pkgdesc='Enables infrared cameras that are not directly enabled out-of-the box.'
+arch=(x86_64)
 url='https://github.com/EmixamPP/linux-enable-ir-emitter'
 license=(MIT)
-arch=(x86_64)
-
+depends=(python python-opencv python-yaml)
+makedepends=(git)
```

Comment:
Why did you remove `gcc` and `make`?

Reply:
gcc and make are part of base-devel and don't need to added to makedepends